vagrant/provisioning/roles/activemq/templates/activemq.service

```diff
@@ -14,6 +14,7 @@ RestartSec=10
 StandardOutput=syslog
 StandardError=syslog
 SyslogIdentifier=activemq
+Environment="JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk"
```

Comment:
Alfresco 6 was compiled with Java 11, hence I need to install both java 11 and Java 8.